### DISCUSSION TITLE
Tailwind v4 beta support?

### DISCUSSION THREAD

**Discussion by xsjcTony** [withastro/roadmap#1068] [Proposal]
### Body

Sorry if this is not the place to ask this, but just wondering is there any plan or timeline to support Tailwind v4 beta? It seems pretty exciting and I want to give it a go ASAP, but not sure if the current official integration works well with it.

If the integration will only be updated until Tailwind v4 stable is out, it's absolutely fine, I only asked this because I didn't find any related posts on this.

Thanks!

**Comment by delucis** [comment]
The current integration doesn't support the beta yet, and probably won't until Tailwind v4 is stable.

But there are some guides out there for setting up Tailwind v4 without that, for example https://astro-tips.dev/tips/tailwind-v4/
